apps/api-documenter/package.json

```diff
@@ -29,6 +29,6 @@
     "@types/js-yaml": "3.9.1",
     "@types/node": "6.0.88",
     "gulp": "~3.9.1",
-    "@types/jest": "~20.0.4"
+    "@types/jest": "~21.1.8"
```

Comment:
Did you test the case when one project fails to build?

Reply:
Yes!